README.md

```diff
@@ -34,3 +34,14 @@ Command line (using [sample.json](/sample.json)):
 ```sh
 cat sample.json | fx-crash-sig
 ```
+
+```sh
+python example.py
+```
```

Comment:
Given that there's an `example.py` module that shows how to do some things, I thought I'd surface that.

To run it, you either have to have `.` on your `PYTHONPATH` or we needed to bump it up to the parent directory. I did the latter figuring it's easier to document.

Let me know if you'd rather we did something different.

Reply:
That's good with me.  The `Example script` link above was broken though.